stable/instana-agent/templates/clusterrole.yaml

```diff
@@ -17,6 +17,7 @@ rules:
 - apiGroups: ["batch"]
   resources:
     - "jobs"
+    - 'cronjobs'
```

Comment:
Inconsistent use of quotes? sorry didn't notice before. But, also, doesn't matter much.

Reply:
ARGH